packages/mikado_reborn/src/components/Avatar/Avatar.vue

```diff
@@ -13,7 +13,16 @@ export default class Avatar extends Vue {
 
   @Prop() private alt!: string;
 
+  @Prop() private size!: string;
+
   @Prop() private loading!: 'lazy' | 'eager';
+
+  get avatarStyle(): { width: string, height: string } {
+    return {
+      height: `${this.size || 5}rem`,
+      width: `${this.size || 5}rem`,
+    };
```

Comment:
DRY (puisque `width` == `height`).
```suggestion
    const width = `${this.size}rem`;
    return { height: width, width };
```